Documentation/dev-guide/interacting_v3.md

```diff
@@ -2,7 +2,7 @@
 
 Users mostly interact with etcd by putting or getting the value of a key. This section describes how to do that by using etcdctl, a command line tool for interacting with etcd server. The concepts described here should apply to the gRPC APIs or client library APIs.
 
-By default, etcdctl talks to the etcd server with the v2 API for backward compatibility. For etcdctl to speak to etcd using the v3 API, the API version must be set to version 3 via the `ETCDCTL_API` environment variable. However note that any key that was created using the v2 API will not be able to be queried via the v3 API.  A v3 API ```etcdctl get``` of a v2 key will exit with 0 and no key data, this is the expected behaviour.
+By default, etcdctl talks to the etcd server with the v3 API. For etcdctl to speak to etcd using the v2 API, the API version may be set to version 2 via the `ETCDCTL_API` environment variable. However note that any key that was created using the v2 API will not be able to be queried via the v2 API.  A v3 API ```etcdctl get``` of a v2 key will exit with 0 and no key data, this is the expected behaviour.
 
 
```

Comment:
Makes sense to me, thanks. I've fixed this doc to make sense for 3.3 users. I reckon it's reasonable to keep the other docs changes because those docs are targeted to people building from git?

Reply:
agreed